tests/integration_tests/style/test_json.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests for JSON linter checks."""
+
+import framework.utils as utils
+import json
+
+
+def test_json_style():
+    """Fail if the repository contains malformed JSON in .json files"""
```

Comment:
You should put a period at the end of the sentence to get rid of the python linter failures.